### PR TITLE
Ready for review

### LINKED ISSUE #16: Translate file "contracts.md"
Notes for contributors, the file can be translated by being aware of and understanding the file you are translating, you must have an understanding of the techniques and the explanation provided by the documentation file, and it must be in Standard Arabic, and the technical terms should be translated as follows:
For example, المتحكمات with the term next to it in English (Controllers)
For example, التهجير and next to it the term in English (Migrations)
And so on
Thank you for your contribution

### PR DESCRIPTION
**Pull request by Mahmoud217TR**
Just need to Check About Using "عقود" or "كونتراكتس" for Laravel Contracts.
And Using "واجهات فَسادس" for Laravel Facades
** Also need a Full Review **

### CODE REVIEW

**Comment by Mahmoud217TR** [comment]
Ready for Review and waiting for approval about using "واجهات كونتراكتس" instead of using "عقود".
Because it was introduced as **Laravel's "contracts"** which means that contracts is a name of the interfaces not actual contracts.

File: contracts.md
Comment (karam-mustafa) [inline contracts.md]: I think we need to remove all the translated words like "كونتراكس و فساد" because there is no real translation of it in the Arabic language, and it cannot be explained in just two words

File: contracts.md
Comment (karam-mustafa) [inline contracts.md]: I don't think that "يعود لذوقم الشخصي" is the best translation, because the development process It is not made according to tastes "اذواق", but rather according to use "استخدام", definitely here what is intended to be used

File: contracts.md
Comment (karam-mustafa) [inline contracts.md]: I liked the word "اسبيان", can you add it please to this [ISSUE](https://github.com/Syrian-Open-Source/arabic-laravel-docs/issues/24) and explain how this word was translated

**Review by karam-mustafa — CHANGES_REQUESTED** [review]

File: contracts.md
Comment (Mahmoud217TR) [reply contracts.md]: ok I'll leave those in English

File: contracts.md
Comment (Mahmoud217TR) [reply contracts.md]: Noted

File: contracts.md
Comment (Mahmoud217TR) [reply contracts.md]: sure

**Comment by Mahmoud217TR** [comment]
I hope this will fix it

**Pull request merged**
